index.html

```diff
@@ -43,5 +43,108 @@ <h2>I'm a web developer</h2>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+             <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+            <div class="grid3">
+                <h2>Profesional Art Printing Data</h2>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul>
+                    <li><p>HTML</p></li>
+                    <li><p>Bootstrap</p></li>
+                    <li><p>Ruby</p></li>
+                </ul>
+                <button class="see-project2">See project</button>
+            </div>
+        </div>
+    </section>
+    <section class="about-section">
+        <div class="about-me">
+            <h1>About me</h1>
```

Comment:
- [ ] Please use only one h1 tag in each of your document. This is best SEO practice. Kindly change this h1 tag to any other heading tag between h2 and h5